.github/workflows/convention-develocity-gradle-plugin-verification.yml

```diff
@@ -69,6 +69,9 @@ jobs:
           - sample: '6.9_and_later'
             version: 'wrapper'
             java: '21'
+          - sample: '6.9_and_later'
+            version: 'release-candidate'
+            java: '21'
```

Comment:
What do you think about renaming these to `gradle-version` and `java-version`, to be more explicit? If you do that, I would also apply a similar change to Maven.

Reply:
Done